packages/css/src/components/mega-menu/mega-menu.scss

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+@import "../../common/breakpoint";
+
+.amsterdam-mega-menu {
+  padding-block: 3.5rem 4.5rem;
+  padding-inline: 1.5rem;
+
+  @media screen and (min-width: $amsterdam-breakpoint-wide) {
+    padding-block: 5rem 7.5rem;
+    padding-inline: 5.24rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-grid__cell {
+  .amsterdam-heading:nth-child(n + 1) {
+    padding-block-start: 3.5rem;
+  }
+}
+
+.amsterdam-mega-menu .amsterdam-unordered-list {
+  --amsterdam-grid-gap: var(--amsterdam-grid-spacious-gap);
```

Comment:
Is this var necessary? This creates a dependency between MegaMenu and Grid. Couldn't we just set the correct column gap in our tokens?

Reply:
No, columns do not work on flex, an alternative would be to size the items as flex: 0 0 50%; and wrap them, but I would advise against that.